### DISCUSSION TITLE
Style file input button

### DISCUSSION THREAD

**Discussion by gageleblanc** [twbs/bootstrap#34761] [Q&A] [answered]
Hello! I have pretty successfully created a layout for the application I am developing in dark mode using minimal custom CSS, but I am running into trouble styling the file input button and haven't been able to find much help... I was wondering if there was a simple way to style the button, since it doesn't seem the btn-* classes effect the browse button. I know in the past I have personally had issues styling the default button, so I wasn't sure if there was maybe some JS I needed to write as well to get it working.

**Comment by mdo** [comment, marked as answer]
https://getbootstrap.com/docs/5.1/forms/form-control/#file-input

**Reply by gageleblanc** [reply]
Apologies, feel a bit dumb now.... didn't even think to take a look IN the css itself... In case anyone else has the more specific issue of coloring the button itself, you can use `.form-control::file-selector-button` in CSS.

**Reply by ffoodd** [reply]
It's also explained in our Forms' docs page. 🙂
